src/Illuminate/Routing/Router.php

```diff
@@ -562,6 +568,36 @@ protected function mergeGroupAttributesIntoRoute($route)
     }
 
     /**
+     * Determine if action is referenced as an array.
+     *
+     * @param  string|array  $action
+     * @return bool
+     */
+    private function actionIsReferencedAsArray($action)
+    {
+        if (! is_array($action)) {
+            return false;
+        }
+
+        list($class, $method) = each($action);
```

Comment:
Could be replaced by `$class = $action[0];` since `$method` is not used afterwards.